qiskit_experiments/library/calibration/fine_frequency_cal.py

```diff
@@ -0,0 +1,134 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Fine frequency calibration experiment."""
+
+from typing import List, Optional
+import numpy as np
+
+from qiskit.providers.backend import Backend
+
+from qiskit_experiments.framework import ExperimentData
+from qiskit_experiments.calibration_management.update_library import BaseUpdater
+from qiskit_experiments.calibration_management import (
+    BaseCalibrationExperiment,
+    BackendCalibrations,
+)
+from qiskit_experiments.library.characterization.fine_frequency import FineFrequency
+
+
+class FineFrequencyCal(BaseCalibrationExperiment, FineFrequency):
+    """A calibration version of the fine frequency experiment."""
```

Comment:
```suggestion
    """A calibration version of the fine frequency experiment.
    
    # section: see_also
        :py:class:`FineFrequency`
    """
```

I added see also section for all calibration experiments. See other class docstrings.

Reply:
Thanks